designer/client/outputs/notify-edit-items.tsx

```diff
@@ -24,6 +24,7 @@ class NotifyItems extends React.Component<Props, State> {
   }
 
   onClickAddItem = (_event: MouseEvent) => {
+    _event.preventDefault();
```

Comment:
`_varname` is a naming convention in ts - if you are not using the variable, but still want to declare the type, you put `_` before it. Previously it was not being used, so that's why it was _event. You should remove the underscore now